pkg/sql/submitter.go

```diff
@@ -164,17 +160,16 @@ func (s *defaultSubmitter) ExecuteTrain(cl *ir.TrainStmt) (e error) {
 }
 
 func (s *defaultSubmitter) ExecutePredict(cl *ir.PredictStmt) (e error) {
-	if e = s.LoadModel(cl.TrainStmt); e == nil {
-		if e = createPredictionTableFromIR(cl, s.Db, s.Session); e == nil {
-			var code string
-			if isXGBoostModel(cl.TrainStmt.Estimator) {
-				code, e = xgboost.Pred(cl, s.Session)
-			} else {
-				code, e = tensorflow.Pred(cl, s.Session)
-			}
-			if e == nil {
-				e = s.runCommand(code)
-			}
+	// NOTE(typhoonzero): model is already loaded under s.Cwd
```

Comment:
Do we need LoadModel anymore? How about deleting it?

Reply:
Done